submissions/RomanSaiko/html-css-popup/index.html

```diff
@@ -0,0 +1,99 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
```

Comment:
And it's better to avoid viewport values that prevent users from resizing documents.

Reply:
And if you use such thing as [Emmet](https://emmet.io/) (for example, it is included in VS Code by default). 
You can type just `!` and basic structure for html page will appear.
There are mane useful shortcuts too.

Reply:
My default Emmet in PhpStorm added viewport attribute, so I didn't pay attention.